README.md

```diff
@@ -1,36 +1,31 @@
 # Telescope.nvim
 [![Gitter](https://badges.gitter.im/nvim-telescope/community.svg)](https://gitter.im/nvim-telescope/community?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
 
-> **Telescope**
-> An arrangement of lenses or mirrors or both that gathers light,
-> permitting direct observation or photographic recording of distant objects.
->
-> — thefreedictionary
-
-![by @glepnir](https://user-images.githubusercontent.com/41671631/100819597-6f737900-3487-11eb-8621-37ec1ffabe4b.gif)
+Gaze deeply into unknown regions using the power of the moon.
 
+## What Is Telescope?
 
-`Telescope.nvim` is a next generation library for creating floating pickers
-with advanced features.  It is written in `lua` and is built on top of latest
-awesome features from `neovim` core.  `Telescope.nvim` is centered around
-modularity so much so that each picker is its own world, (meaning it can be
-configured in isolation from other pickers, such presentation, algorithm,
-mappings ... etc).
+`telescope.nvim` is a highly extendable fuzzy finder over lists.   Built on top of latest
+awesome features from `neovim` core.  Telescope is centered around
+modularity, allowing for easy customization.
```

Comment:
Here (after around modularity) perhaps we need to mention to what extent telescope is modular. Modularity is very general feature that could mean different things.

Reply:
I think its all modular so everything can be swapped out? From the end user, this should be accurate.

Reply:
Maybe we say the sorters, pickers, finders can all be composed modularly?

Reply:
Oh thats it. that should do it